examples/diabetes_example/assets/functions/description.md

```diff
@@ -0,0 +1 @@
+lorem ipsum
```

Comment:
Let's add a description specific to function but only one or two sentences would do the job.